### PR TITLE
Move network probes tests to network package

### PR DESCRIPTION
**Pull request by ormergi**
Signed-off-by: Or Mergi <ormergi@redhat.com>

**What this PR does / why we need it**:
Following #5170 #5167 
Move network related probes tests to network package

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

**Comment by rmohr** [comment]
I think especially the probe tests are tricky to decide where they fit best. We only support networking probes right now, but there exist other  probe types (we think for instance for a long time about guest-agent-probes), further I see most of the complexity with  them on the kubevirt-composition level with kubernetes. That networking needs to work is of course a precondition, but I don't see it as the main complexity-driver on these tests. What do you think?

**Comment by ormergi** [comment]
> I think especially the probe tests are tricky to decide where they fit best. We only support networking probes right now, but there exist other probe types (we think for instance for a long time about guest-agent-probes), further I see most of the complexity with them on the kubevirt-composition level with kubernetes. That networking needs to work is of course a precondition, but I don't see it as the main complexity-driver on these tests. What do you think?

My intent was to move probes tests that related to network, since there are no others kind of probes that we test I moved it as is.
When we will test other kind of probes, e.g guest-agent-probes, we could extract the common code to another package and reuse it for other tests.
I will change the file name and the Describe block to reflect that.
What you think?

**Review by EdDev — COMMENTED** [review]
@rmohr these tests seem to be network related, checking the integration with k8s. They have been treated in the past by the network sig and with the current content we can assign the primary ownership to this sig. Future probes can land on a different owner.
Are you ok with that? Do you have a different sig in mind for these?

@ormergi , I think you need to run make/make-functest(dry-run) so the bazel files will get updated.

File: tests/network/BUILD.bazel
Comment (fgimenez) [inline tests/network/BUILD.bazel:11]: Maybe you could drop the `network` in the name, given that it will be under the `network` subdir? So that the path looks like `tests/network/probes.go`, wdyt?

**Review by fgimenez — COMMENTED** [review]
Looks good, small nit about the file name.

File: tests/network/BUILD.bazel
Comment (ormergi) [reply tests/network/BUILD.bazel:11]: I changed the name to emphasize these are probes tests that related to network,
so when there will be other probes tests it will be clear which file tests a kind of probes.
But like you suggests it could be understood from the path
I will change it

**Review by fgimenez — APPROVED** [review]
/lgtm

File: tests/network/BUILD.bazel
Comment (fgimenez) [reply tests/network/BUILD.bazel]: Awesome, thanks!

**Review by EdDev — APPROVED** [review]
Thank you!

**Comment by oshoval** [comment]
Did you run sig-network lane ? at least once if not couple times ?
I don't see it
/hold

feel free to remove hold once you are ready

**Comment by EdDev** [comment]
> Did you run sig-network lane ? at least once if not couple times ?
> I don't see it, the lane is currently still manual / periodic
> 
> /hold
> 
> feel free to remove hold once you are ready

Ohh, right! We need to see 10 successful runs with it before we merge.

**Comment by ormergi** [comment]
> Ohh, right! We need to see 10 successful runs with it before we merge.

Done, all green 
/unhold

**Pull request merged**
